src/components/clusterDetail/ConsoleModal.tsx

```diff
@@ -30,39 +32,121 @@ type ConsoleModalProps = WebConsoleHintProps & {
   isOpen: boolean;
 };
 
+type ModalExpandableSectionProps = {
+  toggleText: string;
+  className: string;
+  isExpanded?: boolean;
+  onToggle?: () => void;
+  requiredList: string[];
+  optionalList?: string[];
+};
+
+const ModalExpandableSection: React.FC<ModalExpandableSectionProps> = (props) => {
+  const { requiredList, optionalList, ...otherProps } = props;
+  return (
+    <ExpandableSection {...otherProps}>
+      <TextContent>
+        <TextList>
+          <Text component="p">
```

Comment:
React will warn when putting `div`s inside `p`. (`Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.`)

We should use `TextListItem` but PatternFly does not provide list-style-type: none `li` :/

Alternatively, since the content of this is going to be usually copied by the user, why not render the lists in a `<PrismCode />` component?

<TextContent>
  <Text component="h6">Title if needed<Text>
  <PrismCode .... />
  ...

Reply:
PrismCode is an excellent idea!